src/workflow/workflow.js

```diff
@@ -548,20 +550,43 @@ const setInstanceMetadata = async (wfConfig) => {
 };
 
 const sendInitialFunds = async (wfConfig) => {
-  // 7-fund accounts with some initial funds
-  const amount = wfConfig?.instance?.initialFund;
-
-  // if no initialFund is set or initialFund is set to zero, skip this step.
-  if (!amount) {
-    console.log(
-      'no initialFunds was set in workflow. skipping sendInitialFunds!'
-    );
-    return;
-  }
+  // 7 - fund accounts with some initial funds
+  let initialFund = wfConfig?.instance?.initialFund;
 
   const context = getContext();
-  const { startRecordNo, endRecordNo } = context.data;
   const { dryRun } = context;
+  const { startRecordNo, endRecordNo } = context.data;
+  const { api, signingPair } = context.network;
+
+  if (!initialFund.match(initialFundPattern)) {
+    const { calcInitialFund } = (await inqAsk([
```

Comment:
Let's first calculate the funds and then in the question message we can show them the amount and ask them if they  want to set the amount to the calculated initial fund.
It can even show them the human firendly amount (e.g. 1.2 mili KSM) using the fomatBlance utility:

https://github.com/polkadot-js/common/blob/f39820e9c0a93fbc40041ad63a26bc2961721091/packages/util/src/format/formatBalance.spec.ts#L108

https://github.com/polkadot-js/common/blob/f39820e9c0a93fbc40041ad63a26bc2961721091/packages/util/src/format/formatBalance.spec.ts#L84

Reply:
Would be also cool to show them the total fund that is needed (initial funds*number of addresses) and also check the sender's account balance is above the total initial funds to make sure they have enough funds and it won't error out in the middle. it can alert them to fill their account to continue.